contracts/seed/Seed.sol

```diff
@@ -242,13 +239,14 @@ contract Seed {
         require(tokenLocks[msg.sender].fundingAmount > 0, "Seed: zero funding amount");
         Lock storage tokenLock = tokenLocks[msg.sender];
         uint256 amount = tokenLock.fundingAmount;
-        seedRemainder = seedRemainder.add(tokenLock.seedAmount);
+        uint256 feeAmount = (tokenLock.seedAmount).mul(uint256(PPM)).mul(fee).div(PPM100);
```

Comment:
I think we have a fee value in the Lock structure exactly for this purposes, why do we need to calculate it here?

Reply:
yup, I saw it now, have changed it.

Reply:
I think you haven't commited it yet?

Reply:
this feeAmount is in fundingAmount, and we are storing the fee calculated on seedAmount at the Lock.

Reply:
Ok, it sees good, I think it would be cheaper to calculate from feeAmount(only 2 operations are needed there), but really nice anyways